en/index.html

```diff
@@ -1,108 +1,103 @@
-    <meta name="twitter:image" content="//ap.hamakor.org.il/2016/ap_button.png">
-
-    <meta name="description"  content="August Penguin is the annual convention of the Israeli free and open source software community. The convention is organized by the Hamakor NGO. The convention provides an opportunity for a social gathering alongside technical lectures and presentation of initiatives, projects, and other updates related to free and open source software over the past year." />
-    <meta name="keywords" content="אוגוסט פינגווין, תוכנה חופשית, קוד פתוח, עמותת המקור, August Penguin, Free Software, Open Source, HaMakor NPO, ap2016, אפ2016" />
-    <meta name="author"       content="Hamakor - Israeli free and open source software NGO" />
-    <meta property="og:image" content="//ap.hamakor.org.il/2016/ap_button.png" />
-    <meta property="og:title" content="The annual free and open source software convention" />
-    <meta property="og:site_name" content="August Penguin" />
-
-    <link rel="shortcut icon" href="favicon.png" type="image/png" />
-    <title>August Penguin 2016</title>
-    <link rel="stylesheet"    href="../ap2016.css"    type="text/css"  />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-  </head>
-
- <body style="direction: ltr;">
-    <div id="fb-root"></div>
-
-    <div id="header">
-      <div class="hgroup">
-        <h1>August Penguin 2016</h1>
-      </div>
-    </div>
-
-    <div class="section" id="sponsorship">
-      <nav class="language"><a href="../">עברית</a></nav>
-          <h1 class="noMobile">
-        <a href="http://hamakor.org.il/">Hamakor</a> is proud to present:
-      </h1>
-
-      <img src="../ap_button.png" alt="August Penguin 2016 - the annual free and open source software convention" class="noMobile" />
-      <h2>Sponsored by:</h2>
-      <div class="sponsors">
-        Want to sponsor? Please contact <a href="mailto:board@hamakor.org.il">board@hamakor.org.il</a>
-      </div>
-    </div>
-
-    <div class="section" id="content">
-      <a name="location" id="location"></a>
-      <h2>Where and When?</h2>
-      <dl class="location">
-        <dt>
-          When:
-        </dt>
-        <dd>
-          Friday, August 5th, 2016
-        </dd>
-        <dt>
-          Location
-        </dt>
-        <dd>
-          Building 37, Ben Gurion University, Be'er Sheva
-        </dd>
-      </dl>
-
-      <a id="registration" name="registration"></a>
-      <h2>Registration</h2>
-      <div>
-        Will open soon
-      </div>
-
-      <a id="schedule" name="schedule"></a>
-      <div id="schedule-section">
-        <h2>Agenda</h2>
-        <div>
-          Final agenda will be published in July. Want to participate? The call for presentations is open! Please fill in the form below:
+<html class="no-js" lang="en">
```

Comment:
Please DO NOT override the whole file when you are doing just minor changes. This breaks the blame view.